jax/experimental/jet.py

```diff
@@ -12,6 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+r"""Jet is an experimental module for higher-order automatic differentiation
+  that does not rely on repeated first-order automatic differentiation.
+
+  How? Through the propagation of truncated Taylor polynomials.
+  Consider a function :math:`f = g \circ h`, some point :math:`x`
+  and some offset :math:`v`.
+  First-order automatic differentiation (such as :func:`jax.jvp`)
+  computes the pair :math:`(f(x), \partial f(x)[v])` from the the pair
+  :math:`(h(x), \partial h(x)[v])`.
+
+  :func:`jet` implements the higher-order analogue:
+  Given the tuple
+
+  .. math::
+    (h_0, ... h_K) :=
+    (h(x), \partial h(x)[v], \partial^2 h(x)[v], ..., \partial^K h(x)[v,...,v]),
+
+  which represents a :math:`K`-th order Taylor approximation
+  of :math:`h` at :math:`x`, :func:`jet` returns a :math:`K`-th order
+  Taylor approximation of :math:`f` at :math:`x`,
+
+  .. math::
+    (f_0, ..., f_K) :=
+    (f(x), \partial f(x)[v], \partial^2 f(x)[v], ..., \partial^K f(x)[v,...,v]).
+
```

Comment:
the PDF you linked in the other comment discusses normalised Taylor coefficients (i.e. f_k / k!), but as far as I am aware, `jet` does not work with normalised coefficients. Right? 

Because if it does, I should update this docstring! :)

Reply:
> the PDF you linked in the other comment discusses normalised Taylor coefficients (i.e. f_k / k!), but as far as I am aware, jet does not work with normalised coefficients. Right?

Hah, I forget! Thanks for checking. It would be good to make sure the docstring agrees with the code. If you've looked at the code recently, I trust your evaluation as to which it uses!